lib/smart_answer/calculators/benefit_calculator.rb

```diff
@@ -0,0 +1,62 @@
+module SmartAnswer::Calculators
+  class BenefitCalculator
+    attr_accessor :single_couple_lone_parent, :postcode
+
+    def initialize
+      @benefits = Hash.new(0)
+    end
+
+    def claim(benefit, amount)
+      @benefits[benefit] = amount
+    end
+
+    def amount(benefit)
+      @benefits[benefit]
+    end
+
+    def total_benefits
+      @benefits.values.inject(0) {|sum, value| sum + value }
+    end
+
+    def benefit_cap
+      single_couple_lone_parent == 'single' ? benefit_cap_rate(:single) : benefit_cap_rate(:couple)
+    end
```

Comment:
This method seems to implicitly assume that the `couple` and `parent` rates are the same. While this is true at the moment the `benefit_caps.yml` file suggests it may not always be true.